app/src/main/java/no/usn/mob3000/ui/screens/info/docs/CreateDocumentationScreen.kt

```diff
@@ -86,7 +89,10 @@ fun CreateDocumentationScreen(
             Button(
                 onClick = onSaveDocumentationClick,
                 modifier = Modifier.fillMaxWidth(),
-                colors = ButtonDefaults.buttonColors(DefaultButton)
+                colors = ButtonDefaults.buttonColors(
+                    containerColor = MaterialTheme.colorScheme.primary, // Bruker primary fargen fra temaet
+                    contentColor = MaterialTheme.colorScheme.onPrimary // Bruker fargen for tekst (content) på primary
+                )
```

Comment:
If you are using AI, you need to APA7 reference the conversation and also have the conversation browsable for the teachers perusal. Preferably, do not use AI. Additionally, these comments serve no purpose, they literally just repeat what the code already says explicitly.